apps/shade/src/components/ui/skeleton.tsx

```
@@ -1,14 +1,29 @@
-            className={cn('animate-pulse rounded-md bg-primary/10', className)}
-            {...props}
-        />
+        <span className={containerClassName}>
+            {Array.from({length: count}).map(() => (
+                <>
+                    <span
+                        key={`skeleton-${crypto.randomUUID()}`}
+                        className={cn('inline-flex w-full leading-none animate-pulse rounded-[2px] bg-primary/10', className)}
+                        {...props}
+                    >&zwnj;</span>
+                    <br />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)